src/App.js

```diff
@@ -250,7 +258,7 @@ export default class App extends Component {
         return body;
     };
 
-    getUsersAPI = async () => {
+    getUsersAPI = async (id) => {
```

Comment:
The variable 'id' isn't used in this function body. Also, we can remove this function and replace its uses with the function `getUserAPI` below because they essentially do the same things.